react/AppLinker/native.spec.js

```diff
@@ -0,0 +1,36 @@
+import { generateUniversalLink, getUniversalLinkDomain } from './native'
+describe('native functions', () => {
+  it('should generate the universalink', () => {
+    const universalLink = generateUniversalLink({
+      slug: 'drive',
+      nativePath: '/files/1',
+      fallbackUrl: 'https://drive.cozy.tools/files/1'
+    })
+    const endLink =
+      getUniversalLinkDomain() +
```

Comment:
I would suggest not using a tested function to generate the *expected* result. I think hardcoding the domain here would be ok, no?